### PR TITLE
refactor(prettier): Reverse .prettierignore to use as breakdown list

### PR DESCRIPTION
**Pull request by queengooborg**
This reverses the Prettier ignore, so that instead of ignoring all Markdown and making exceptions for folders we have performed a pass on, we only ignore the folders that are YET to be formatted.  This allows us to use the `.prettierignore` as a breakdown list of what still needs to be performed.

This also places slashes at the beginning of each ignore statement to ensure more explicit matching of files (see https://github.com/mdn/content/pull/20674/files#r1096228284 for more context).

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -3,22 +3,20 @@
...
-!files/en-us/web/performance/**/*.md
-!files/en-us/web/privacy/**/*.md
-!files/en-us/web/security/**/*.md
-!files/en-us/web/webdriver/**/*.md
-!files/en-us/web/xml/**/*.md
-!files/en-us/web/xpath/**/*.md
-!files/en-us/web/xslt/**/*.md
+# A full pass on all Markdown files is being performed, the following folders
+# still need a full pass:
+/files/en-us/games/**/*.md
```
Comment (nschonni) [inline .prettierignore:8]: I had cleaned this one up awhile ago, and I think it's still good

Comment (queengooborg) [reply .prettierignore]: Looks like it is, yes!

File: .prettierignore [hunk 2]
```diff
@@ -3,22 +3,19 @@
...
+/files/en-us/web/html/**/*.md
+/files/en-us/web/javascript/**/*.md
+/files/en-us/web/mathml/**/*.md
+/files/en-us/web/progressive_web_apps/**/*.md
+/files/en-us/web/svg/**/*.md
+/files/en-us/web/text_fragments/**/*.md
+/files/en-us/web/web_components/**/*.md
+
+# The /web/api folder is so large, it has been divided into multiple parts
+/files/en-us/api/**/*.md
```
Comment (nschonni) [inline .prettierignore:20]: Noticed this because I cherry-picked this over to the CI PR
```suggestion
/files/en-us/web/api/**/*.md
```

**Review by bsmth — APPROVED** [review]
Looks good, thanks 👍🏻

**Pull request merged**